code/obj/machinery/vortex_wraith.dm

```diff
@@ -6,7 +6,7 @@
 	anchored = 1
 	density = 1
 	_health = 25
-	var/list/mob_list = list()
+	var/list/critter_list = list()
```

Comment:
type this?

Reply:
Done so, it is now var/list/obj/critter/critter_list